src/model.py

```diff
@@ -247,8 +248,11 @@ def add_drop_filename(self, filename: str) -> None:
     def add_mutex(self, mutex: str) -> None:
         self.mutexes.append(mutex)
 
-    def add_email(self, email: str) -> None:
```

Comment:
It'll be a backward incompatible change, right? If this was a bigger lib I'd prefer to just make this deprecated for now (and alias for emails_to). But it's not, and there are  no official users on github. Still, might be worth checking with our slack *friends* if that's OK with them. And we should call next version 2.0 i guess.